### DISCUSSION TITLE
column folded and can't restore

### DISCUSSION THREAD

**Discussion by wonb168** [dbeaver/dbeaver#35541] [Q&A] [answered]
### Description

The execution log interface of my DBeaver originally had columns for execution duration and the number of affected rows, 
but I accidentally dragged and folded them together，
 and now I can’t restore them. 
Uninstalling and reinstalling doesn’t work. 
What should I do?
![image](https://github.com/user-attachments/assets/aa60654c-7f0f-41ba-98a5-7fe9baefb7e6)

### DBeaver Version

24.1.4

### Operating System

macos 14.6.1

### Database and driver

pg jdbc 42.7

### Steps to reproduce

dragged the column and folded them together，
 and then can’t restore them.

### Additional context

have not

**Comment by E1izabeth** [comment]
Please try repeating my actions on the video


https://github.com/user-attachments/assets/e66bd626-c548-48da-820e-02ee698b1acc


Maybe we need some `Reset` action for this view

**Comment by wonb168** [comment]
cannot drag to restore，and my action in following gif：
![PixPin_2024-08-26_10-09-28](https://github.com/user-attachments/assets/aa44817c-6a91-4ac6-81e5-67f0fb33afae)
mypc：mac mini m2，arm

**Comment by E1izabeth** [comment, marked as answer]
OK
Then you can try resetting UI settings
1.
![image](https://github.com/user-attachments/assets/e8b7d621-3d4b-4019-9049-5d9b79da69c6)
2.
![image](https://github.com/user-attachments/assets/41660661-b03e-47c2-b159-cf0d2cceca39)
